frontend/app_flowy/lib/workspace/presentation/widgets/edit_pannel/edit_pannel.dart

```diff
@@ -11,18 +11,14 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 class EditPannel extends StatelessWidget {
   late final EditPannelContext editContext;
   final VoidCallback onEndEdit;
-  EditPannel(
-      {Key? key,
-      required Option<EditPannelContext> context,
-      required this.onEndEdit})
-      : super(key: key) {
+  EditPannel({Key? key, required Option<EditPannelContext> context, required this.onEndEdit}) : super(key: key) {
```

Comment:
Please add a comma "," after "this.onEndEdit" and then reformat the code. This will make it match the way it was.

Reply:
Will recheck but dont remember touching this file 🤔